README.md

```diff
@@ -151,6 +169,11 @@ pipenv run coverage run --source=updater -m unittest -v
 pipenv run coverage report
 ```
 
+Updating deps in setup.py:
+```bash
+pipenv-setup sync -d
```

Comment:
Love this! Was thinking to myself "ugh will I have to manually sync the locked dependencies?!" this module rules.

Reply:
I'll be honest I hadn't worked much with pipenv or Pipfiles and once I thought "oh right, how do I sync these" I got really happy that problem was solved haha